### PR TITLE
Update README to Asciidoc 

### PR DESCRIPTION
**Pull request by Adyel**
Check the README and let me know what do you think. I have added Table of Content for Better Navigation

### CODE REVIEW

**Comment by mominul** [comment]
@Adyel Bullets points এর মধ্যে এত spacing কেন?

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+:toc:
+:toc-placement!:
+
+//HTML for formating the logo
+
+++++
+<p align="center">
+<img src="https://github.com/OpenBangla/OpenBangla-Keyboard/raw/master/data/128.png">
+</p>
+++++
```
Comment (mominul) [inline README.adoc]: How about using [image](http://gist.asciidoctor.org/?github-asciidoctor%2Fdocgist%2F%2Fgists%2Fimages.adoc#images) with [align attribute](https://asciidoctor.org/docs/user-manual/#positioning-attributes)?

Comment (Adyel) [reply README.adoc]: It does not work on github

**Comment by mominul** [comment]
Is this PR ready for merging? I'd want to merge this.

**Comment by Adyel** [comment]
@mominul Yeah, It is ready

**Pull request merged**